src/components/Content/Content.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { switchError } from '../../utils/switchError';
+import { UserSelector } from '../UserSelector';
+import { ErrorNotification } from '../ErrorNotification';
+import { PostsList } from '../PostsList';
+import { Loader } from '../Loader';
+import { Post } from '../../types/Post';
+import { Error } from '../../types/Error';
+
+type Props = {
+  posts: Post[];
+  postId: number | null;
+  selectedUserId: number | null;
+  error: Error | null;
+  isLoading: boolean;
+  onPost: (id: number | null) => void;
+  onSelectUser: (userId: number) => void;
+  onError: (error: Error | null) => void;
+};
+export const Content: React.FC<Props> = ({
+  posts,
+  postId,
+  selectedUserId,
+  error,
+  isLoading,
+  onPost,
+  onSelectUser,
+  onError,
+}) => (
+  <div className="tile is-parent">
+    <div
+      className="
+        tile
+        is-child
+        box
+        is-success"
+    >
+      <UserSelector
+        selectedUserId={selectedUserId}
+        onSelectUser={onSelectUser}
+        onError={onError}
+      />
+      <div className="block" data-cy="MainContent">
+        {!selectedUserId
+          && !error
+          && (
+            <p data-cy="NoSelectedUser">
+              No user selected
+            </p>
+          )}
+
+        {error
+          && switchError(error) === 'PostsLoadingError'
+          && (
+            <ErrorNotification
+              error={error}
+            />
+          )}
+
+        {!posts?.length
+          && selectedUserId
+          && error === Error.NO_POSTS
+          && (
+            <ErrorNotification
+              error={error}
+            />
+          )}
+
+        {isLoading
+          ? <Loader />
+          : posts.length > 0
```

Comment:
pls [see falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
          : !!posts.length 
```